docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -466,6 +466,18 @@ So if you want to deploy your application somewhere, you need to copy the _runne
 
 NOTE: Before running the application, don't forget to stop the hot reload mode (hit `CTRL+C`), or you will have a port conflict.
 
+=== Using fast-jar
+
+The previous section assumes that the configuration property `quarkus.package.type` has either not been configured explicitly,
+or that it has been set to `legacy-jar` (in `application.properties` or any of the other supported configuration sources).
+When the property has been set to `fast-jar` (which will become the default soon), then the result of executing
```

Comment:
Would it make sense to add two sentences from the ``RunnerClassLoader``'s javadoc to explain why it's a good idea to try the `fast-jar` right now! ;-)

Reply:
That might makes sense, but I wouldn't really know where to add it.

Furthermore, if we want users to try something now, I am not sure the docs are the best place to encourage them to do that, as only people who are very new to Quarkus and are just now reading the docs are going to see that call to action

Reply:
> Furthermore, if we want users to try something now, I am not sure the docs are the best place to encourage them to do that, as only people who are very new to Quarkus and are just now reading the docs are going to see that call to action

Yeah, my point is that if someone read the docs (probably by accident) and then noticed that there is a `fast-jar` which is... probaby _fast_ and that's about it ;-).

Reply:
I'm still ambivalent because the reason why `fast-jar` is fast is too technical for the getting started places we have documented it in :)

Reply:
Hm, I think that we should either mention it && describe the advantage or keep it hidden ;-).

Reply:
@gsmet WDYT?

Reply:
Well, AFAIU, the idea was to make `fast-jar` the default for 1.12. Given we're in the 1.12 release cycle. I would start documenting it this way.

That would solve your issue :).

Note that it's going to require a major upgrade of all the guides too.

I was planning to start working on this after the 1.11.1.Final release. I think we could start going into this direction.

Reply:
Seems like I have a ton of things piled up, so if anyone wants to update this, feel free to push into my PR

Reply:
I stole a few minutes and updated the doc. Hopefully you'll find it better now :)